src/Bicep.Cli/CliResources.resx

```diff
@@ -122,6 +122,9 @@
 You may need to fix warnings and errors in the generated bicep file(s), or decompilation may fail entirely if an accurate conversion is not possible.
 If you would like to report any issues or inaccurate conversions, please see https://github.com/Azure/bicep/issues.</value>
   </data>
+  <data name="SymbolicNamesDisclaimerMessage" xml:space="preserve">
+    <value>WARNING: Symbolic name support in ARM is experimental, and should be enabled for testing purposes only. Do not enable this setting for any production usage, or you may be unexpectedly broken at any time!</value>

```

Comment:
>may [](http://example.com/codeflow?start=175&length=3)

will be?

Reply:
I don't want to let anyone down by making false promises!